samcli/commands/local/lib/swagger/reader.py

```diff
@@ -152,7 +154,7 @@ def _download_swagger(self, location):
         """
 
         if not location:
-            return
```

Comment:
Are we just being explicit? or is this the norm with Python3?

Reply:
This falls under pylint's inconsistent-return-statements (R1710), see http://pylint.pycqa.org/en/latest/technical_reference/features.html for more details.

I ended up just updating everything to make this go away. We could instead ignore this and revert the commit with all these types of changes.